static/js/board.js

```diff
@@ -1,117 +1,9 @@
 /*
 Main Futureboard general JS components :
-Contains socket configuration, listens on board controls,
-Executes functions when document is ready(auto scroll, carousel interval).
 */
 
 // Socket.io configuration
 var socket = io();
```

Comment:
This file got pretty skinny, is it being included separately to make it more explicit that other files are using sockets? May not be too bad to toss this into the html template (more clear that it's specifically for other files to use)